R/utils.R

```diff
@@ -148,3 +148,16 @@ as_friendly_type <- function(type) {
 paste_classes <- function(x) {
   paste(class(x), collapse = "/")
 }
+
+paste_line <- function (...) {
+  paste(chr(...), collapse = "\n")
+}
+
+check_dots_empty_s3_consistency <- function(...) {
```

Comment:
This will move to ellipsis?

Reply:
For ellipsis the message should probably be:

```r
      "`...` is not empty.",
      "These dots only exist to allow future extensions and should be empty."
```

Here the message is a bit different because it's not about extensions. Probably doesn't matter much though.